whylog/config/__init__.py

```diff
@@ -103,6 +114,46 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def is_free_parser_name(self, parser_name):
+        return parser_name not in self._parsers
+
+    def _is_unique_parser_name(self, parser_name, black_list):
+        return self.is_free_parser_name(parser_name) and (parser_name not in black_list)
+
+    def propose_parser_name(self, line, regex_str, black_list):
+        building_words = AbstractConfig._get_building_words(line, regex_str)
+        if not building_words:
+            return uuid.uuid4()
```

Comment:
I am not sure if we want to show user parser name like:  fa78d298-4b99-450f-a6f3-080f74dfaab0. I would be concerned while seeing that. Maybe something like: parser_name1, parser_name2? That would be easier to remember if needed, easier to search (maybe Whylog would support alphabetical parser name searching or scrolling).